test/unit/resource-mapper-test.js

```diff
@@ -604,7 +606,7 @@ describe('ResourceMapper', () => {
 
     itMapsFile(mapper, 'a file with the port',
       {
-        path: `${rootPath}space/foo.html`,
+        path: `${rootPath}example.org/space/foo.html`,
         hostname: 'example.org'
       },
       {
```

Comment:
Does not have a negative test that checks the error when file path does not start with hostname.

Reply:
We also might want a negative test in the case `includeHost` is false (first block of tests), that it does not strip of any hostname.

Reply:
There are negative tests that don't strip off the hostname. I'll add a new one that purposely throws the error.

Reply:
👍